examples/systemcalls/systemcalls.c

```diff
@@ -16,6 +24,12 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
+    int status = system(cmd);
+
+    if (status == -1) {
```

Comment:
@GautamaGandhi see the return values at https://man7.org/linux/man-pages/man3/system.3.html, I think there are other possible error return values other than -1

Reply:
Thank you, Dan. I missed the other return values. I will follow this from the next assignment onwards.